includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -1721,4 +1722,56 @@ private function validate_selected_payment_method_type( $payment_information, $b
 			);
 		}
 	}
+
+	/**
+	 * Add a new Stripe payment method via the My Account > Payment methods page.
+	 *
+	 * This function is called by @see WC_Form_Handler::add_payment_method_action().
+	 *
+	 * @return array
+	 */
+	public function add_payment_method() {
+		try {
+			if ( ! is_user_logged_in() ) {
+				throw new WC_Stripe_Exception( 'No logged-in user found.' );
+			}
+
+			// phpcs:ignore WordPress.Security.NonceVerification.Missing
```

Comment:
I'm assuming we can ignore the nonce here because its already checked by WooCommerce core in `add_payment_method_action()`? 

Is that correct?

Reply:
Yeah that's correct. Woo checks the nonce here https://github.com/woocommerce/woocommerce/blob/7cbf7a344237b8b8cbbded8c913881fd0ff483f4/plugins/woocommerce/includes/class-wc-form-handler.php#L486-L488 before it later calls `$gateway->add_payment_method();`